modules/tooltip/react/README.md

```diff
@@ -81,25 +41,37 @@ class TooltipExample extends React.Component<{}, TooltipExampleState> {
 
 ### Required
 
-> None
+#### title: string | React.ReactNode
+
+> If present, this will switch the Tooltip into a simpler to use mode where the Tooltip component
+> can wrap a target and `title` is the contents of the tooltip. This should be a string in most
```

Comment:
Isn't this only determined by which component you use? Omitting `title` will not give you the old behavior (and the type requires `title` anyway)

```suggestion
> The text contents of the tooltip. This should be a string in most
```

Reply:
Ah, yes. Remnants of wip